routing/pathfind.go

```diff
@@ -722,6 +721,24 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 		amountToSend := toNodeDist.netAmountReceived +
 			lnwire.MilliSatoshi(inboundFee)
 
+		// Check if accumulated fees would exceed fee limit when this
+		// node would be added to the path.
+		totalFee := int64(amountToSend) - int64(amt)
+
+		log.Trace(newLogClosure(func() string {
```

Comment:
This is now `lnutils.NewLogClosure`

Reply:
that was very strange, I could build the same commit on two different machines, maybe a github caching thing